pipelines/SpeciesTreeFromBusco/main.nf

```diff
@@ -244,15 +244,15 @@ process buscoAnnot {
 *@output path to cDNA fasta
 */
 process runGffread {
-    label 'rc_4gb'
+    label 'rc_4Gb'
     input:
         tuple path(busco_annot), path(genome)
     output:
         path "cdna/*"
     script:
     """
     mkdir -p cdna
-    ${params.gffread_exe} -x cdna/$genome -g $genome $busco_annot
+    ${params.gffread_exe} -w cdna/$genome -g $genome $busco_annot
```

Comment:
As discussed in person, this was reverted because the original change to `-x` is no longer needed.